SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/RPCService.cpp

```diff
@@ -0,0 +1,339 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/RPCs/RPCService.h"
+
+namespace SpatialGDK
+{
+RPCService::RPCService(const FSubView& InRemoteSubView, const FSubView& InLocalAuthSubView)
+	: RemoteSubView(&InRemoteSubView)
+	, LocalAuthSubView(&InLocalAuthSubView)
+{
+}
+
+void RPCService::AddRPCQueue(FName QueueName, RPCQueueDescription&& Desc)
+{
+	Queues.Add(QueueName, MoveTemp(Desc));
```

Comment:
Could move queue name?

Reply:
I could. FName are interned string though, they should be cheap to copy.